filestack/Filelink.php

```diff
@@ -38,6 +49,9 @@ public function __construct($handle, $api_key = '', $security = null,
         }
 
         $this->http_client = $http_client; // CommonMixin
+
+        $cname = filter_var($cname, FILTER_VALIDATE_DOMAIN, FILTER_FLAG_HOSTNAME);
+        $this->cname = $cname ? $cname : null;
```

Comment:
As it stands, this code will silently ignore poorly formatted domains. Should it raise an exception or call `trigger_error` instead?